pyramid_rpc/jsonrpc.py

```diff
@@ -363,9 +371,11 @@ def add_jsonrpc_endpoint(config, name, *args, **kw):
     kw['jsonrpc_batched'] = True
     kw['renderer'] = null_renderer
     config.add_view(batched_request_view, route_name=name,
-                    permission=NO_PERMISSION_REQUIRED, **kw)
+                    permission=NO_PERMISSION_REQUIRED,
+                    require_csrf=require_csrf, **kw)
     config.add_view(exception_view, route_name=name, context=Exception,
-                    permission=NO_PERMISSION_REQUIRED)
+                    permission=NO_PERMISSION_REQUIRED,
+                    require_csrf=require_csrf)
```

Comment:
This is an exception view, it should always have `require_csrf=False`.

Also the `JsonRpcError` exception view in the `includeme` should also have `require_csrf=False`.